version.txt

```diff
@@ -1 +1 @@
-0.4.7.20240930
+0.4.7.20241005
```

Comment:
Since you are adding to an interface, e.g. the Zstd enumerated constant, you should bump the patch version.